applications/ipc_radio/sysbuild.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+SB_CONFIG_PARTITION_MANAGER=n
```

Comment:
Should it be disabled for all platforms?

Reply:
It's now disabled in nrf54h20 cpurad in sample.yaml. The global disable will be merged in this PR https://github.com/nrfconnect/sdk-nrf/pull/13153 .